graf2d/postscript/inc/TPDF.h

```diff
@@ -110,7 +112,7 @@ class TPDF : public TVirtualPS {
    Double_t XtoPDF(Double_t x);
    Double_t YtoPDF(Double_t y);
 
-   ClassDefOverride(TPDF, 0);  //PDF driver
+   ClassDefOverride(TPDF, 1);  //PDF driver
```

Comment:
Version 1 is not necessary. `TPDF` is not stored. And if - static members are not involved.

Reply:
One should keep version `0`

Reply:
ok ...